planning/route_handler/src/route_handler.cpp

```diff
@@ -573,44 +573,26 @@ lanelet::ConstLanelets RouteHandler::getLaneletSequenceUpTo(
 
   lanelet::ConstLanelet current_lanelet = lanelet;
```

Comment:
not sure how many copies occur normally, but wouldn't it be better to have this as a pointer? so the copy in line 574 is not that expensive?

Reply:
I do not see an easy way to use pointers in this case. In the loop we have a temporary `prev_lanelet` that will need to be copied (or moved) anyway.
From my profiling these copies have negligible impact on performance so I think it is okay to ignore them.